modules/olm-installing-opm.adoc

```diff
@@ -0,0 +1,67 @@
+// Module included in the following assemblies:
+//
+// * operators/olm-managing-custom-catalogs.adoc
+
+[id="olm-installing-opm_{context}"]
+= Installing `opm`
+
+To install the `opm` CLI tool on your workstation:
+
+.Prerequisites
+
+* `podman` version 1.4.4+
+
+.Procedure
+
+ifdef::openshift-enterprise,openshift-webscale[]
+. Set the `REG_CREDS` environment variable to the file path of your registry
+credentials for use in later steps. For example, for the `podman` CLI:
++
+----
+$ REG_CREDS=${XDG_RUNTIME_DIR}/containers/auth.json
+----
+
+. Authenticate with `registry.redhat.io`:
++
+----
+$ podman login registry.redhat.io
+----
+endif::[]
+
+. Extract the `opm` binary from the Operator Registry image and copy it to your
+local file system:
++
+ifdef::openshift-origin[]
+----
+$ oc image extract quay.io/openshift/origin-operator-registry:4.5.0 \
+    --path /usr/bin/opm:. \
+    --confirm
+----
+endif::[]
+ifdef::openshift-enterprise,openshift-webscale[]
+----
+$ oc image extract registry.redhat.io/openshift4/ose-operator-registry:v4.4 \
+    --path /usr/bin/opm:. \
+    --confirm
+----
+endif::[]
+
+. Make the binary executable:
++
+----
+$ chmod +x ./opm
+----
+
+. Place the file anywhere in your `PATH`, such as `/usr/local/bin/`.
++
+----
+$ sudo mv ./opm /usr/local/bin/
+----
+
+. Verify the client was installed correctly:
++
+----
+$ opm version
+
+Version: version.Version{OpmVersion:"1.12.3", GitCommit:"", BuildDate:"2020-06-17T09:40:54Z", GoOs:"linux", GoArch:"amd64"}
```

Comment:
most recent version is 1.12.7

Reply:
This is the version I got when pulled out of the 4.5.0 operator-registry image.